doc/source/whatsnew/v0.21.0.txt

```diff
@@ -385,3 +385,4 @@ Other
 - Bug in :func:`eval` where the ``inplace`` parameter was being incorrectly handled (:issue:`16732`)
 - Bug in ``.isin()`` in which checking membership in empty ``Series`` objects raised an error (:issue:`16991`)
 - Bug in :func:`unique` where checking a tuple of strings raised a ``TypeError`` (:issue:`17108`)
+- Bug in :func:`to_datetime` that cannot use tz-aware origin kwarg (:issue:`16842`)
```

Comment:
I don't think this really relfects the issues, as it sounds like it will now work (while the error message was just clarified)